### PR TITLE
ONEUP-4008: 1UP team nodejs project template

### PR DESCRIPTION
**Pull request by yolch-yolchyan**

### CODE REVIEW

File: workflow-templates/up-js-build-template.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          echo "SERVICE_NAME=$SERVICE_NAME" >> $GITHUB_ENV
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.UP_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.UP_AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
+
+      - name: Build - NodeJS - Install, Build and Test
```
Comment (catalin-cretu) [inline workflow-templates/up-js-build-template.yml:48]: I think we don't need this step. The `Setup Node.JS` step is also not required.

I've looked at 2 backend nodejs builds and they run the build phases through the Dockerfile. So we should only need to run that.

File: workflow-templates/up-js-build-template.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# This template is intended to make it fast to migrate 1UP Team's NodeJS projects into GitHub Actions successfully
+# Please consider you might need to tweak a few commands to make your project migration into GitHub Actions successfully
+
+# Make sure SERVICE_NAME is generated correctly in the step: Setup service name
+# Add the following secrets from the repository settings: UP_AWS_ACCESS_KEY_ID, UP_AWS_SECRET_ACCESS_KEY, UP_SLACK_WEBHOOK_URL
+
+# npm commands could be modified to npm if required (e.g. to yarn)
+
+name: 1UP - Build NodeJS project
```
Comment (catalin-cretu) [inline workflow-templates/up-js-build-template.yml:9]: I think we should mention that this build file is designed for  a backend nodejs project

File: workflow-templates/up-js-build-template.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+      - name: Build - Docker
+        run: |
+          DOCKER_IMAGE="${{ env.ECR_REPOSITORY }}/${{ env.SERVICE_NAME }}:${{ env.IMAGE_TAG }}"
+          docker build -t $DOCKER_IMAGE .
+          echo "DOCKER_IMAGE=$DOCKER_IMAGE" >> $GITHUB_ENV
+
+      - name: Release - configure AWS ECR credentials [on master]
+        if: github.ref == 'refs/heads/master'
+        id: login-ecr
```
Comment (aramin) [inline workflow-templates/up-js-build-template.yml:62]: is it needed?

Comment (yolch-yolchyan) [reply workflow-templates/up-js-build-template.yml:62]: Yep, if we want to push the docker image to our ECR.

Comment (yolch-yolchyan) [reply workflow-templates/up-js-build-template.yml:62]: Ahh, ok. A good catch, we don't need it

File: workflow-templates/up-js-build-template.yml (hunk 1 above)
Comment (yolch-yolchyan) [reply workflow-templates/up-js-build-template.yml:48]: Some services have this step (oembed). I think we can leave it and if someone doesn't need it - they just can remove it. Removal should be easier than adding.

**Pull request merged**